src/backend/src/modules/cssAPI/role/controller.ts

```diff
@@ -41,6 +47,33 @@ export const getKCRole = errorWrapper(async (req: Request, res: Response) => {
   res.json(await getRole(role));
 });
 
+/**
+ * @method GET
+ * @param user - The user name to search.
+ * @route /cssAPI/getUser
+ * @protected Requires "playground-admin"
+ */
+export const getKCUser = errorWrapper(async (req: Request, res: Response) => {
+  if (DEBUG)
+    console.info("DEBUG: getKCUser controller in modules/cssAPI called.");
+
+  const user: IDIRUserQuery = {
+    firstName: req.query.user as string,
```

Comment:
```suggestion
    firstName: req.query.firstName as string,
```
Do a guard statement before this to make sure firstName exists on the request